voxelgpt.py

```diff
@@ -11,27 +11,27 @@
 
 import fiftyone as fo
 
-from links.query_moderator import moderate_query
-from links.dataset_schema_handler import query_schema
-from links.query_intent_classifier import classify_query_intent
-from links.docs_query_dispatcher import run_docs_query, stream_docs_query
-from links.computer_vision_query_dispatcher import (
+from voxelgpt.links.query_moderator import moderate_query
```

Comment:
Unfortunately we can't use this import style here because the `voxelgpt.py` module is also intended for [direct usage via Python](https://github.com/voxel51/voxelgpt?tab=readme-ov-file#using-voxelgpt-in-python), in which case the code is not running as a package.

That said, the changes in this module aren't actually required right? Relative imports should be fine.